app/controllers/general.php

```diff
@@ -495,7 +495,7 @@ function router(App $utopia, Database $dbForConsole, SwooleRequest $swooleReques
                 ]);
 
                 $role = Auth::USER_ROLE_APPS;
-                $scopes = \array_merge($roles[$role]['scopes'], $key->getAttribute('scopes', []));
+                $scopes = \array_merge($roles[$role]['scopes'], $key->getAttribute('scopes', []), ['public']);
```

Comment:
API key authenticated clients need `public` scope in order to use account APIs.

Thoughts @christyjacob4 ?